### ISSUE TITLE
[Slicer] Hide button for control points in the SmartEdit/DeepGrow section

### ISSUE THREAD

**Issue by MattTheCuber** [Project-MONAI/MONAILabel#1463]
**Is your feature request related to a problem? Please describe.**
It is annoying to paint with colored dots on the images.

**Describe the solution you'd like**
A simple hide button to toggle the visibility of these control points.

**Describe alternatives you've considered**
None

**Additional context**
Open 3D Slicer, open the MONAI Label module, expand the SmartEdit/DeepGrow section, add some control points. Once you have done this, it is hard to see under the control points if you wanted to view/paint the images under it.

Something like this:
Before:
![vncviewer_R9tx4XmaY6 - Copy](https://github.com/Project-MONAI/MONAILabel/assets/32849887/5ddf435a-a977-454f-a78b-a35acb70b3e5)
After:
![vncviewer_R9tx4XmaY6](https://github.com/Project-MONAI/MONAILabel/assets/32849887/deb5df06-835a-48ed-b3ef-4d1a3efb6922)

**Comment by tangy5** [comment]
Maybe the control points size is too large in the UI, thanks for suggesting. We can make some adjustment in the future.

**Comment by MattTheCuber** [comment]
Adding a hide function would be better in my opinion. That way you can run it, validate, then clear labels and add points if you need to.

**Comment by tangy5** [comment]
> Adding a hide function would be better in my opinion. That way you can run it, validate, then clear labels and add points if you need to.

Thanks, good idea, as a public and open sourced tool,  if convinient, it would be great you can create a PR to adjust the buttons, UI, etc, to meet your demands, I think it's also useful to other users. 

Thank again.

**Comment by MattTheCuber** [comment]
Updated description with screenshot concept

**Comment by lassoan** [comment]
Yes, depending on the markup options, the control points may indeed occlude underlying image content. Instead of hiding the markup completely, there are several options that you can use to make the points less obtrusive. For example, you can set:
- opacity
- glyph size: you can choose to define an absolute size in millimeters; or make it relative to the screen size so that if you zoom in then the size gets relatively smaller
- glyph type: crosshair, vertex, etc. does not occlude as much as a sphere/circle

**Comment by MattTheCuber** [comment]
Those are good options too. For ease of use, I guess it would make sense to implement both an opacity change and a way to hide them.
